### ISSUE TITLE
Deprecation on eslint-config-standard-jsx

### ISSUE THREAD

**Issue by msandrini** [zouhir/eslint-config-standard-preact#15]
Hi! It seems this package uses `eslint-config-standard-jsx` but I've come across this warning:
```
(node:48484) [ESLINT_LEGACY_OBJECT_REST_SPREAD] DeprecationWarning: The 'parserOptions.ecmaFeatures.experimentalObjectRestSpread' option is deprecated. Use 'parserOptions.ecmaVersion' instead. (found in "node_modules/eslint-config-standard-jsx/index.js")
```
But, more than that, it seems that the aforementioned package itself is meant to be deprecated: 
https://github.com/standard/eslint-config-standard-jsx/issues/21

Keep up the good work! Thanks!
